src/content/docs/guides/send-test-payments.mdx

```diff
@@ -5,26 +5,26 @@ title: Send test payments
 import { Steps, Icon } from '@astrojs/starlight/components'
 import { LinkOut, StylishHeader } from '@interledger/docs-design-system'
 
-Use the Interledger Web Monetization browser extension to send payments to a web monetized site without using real money. 
+Use Interledger Foundation's Web Monetization browser extension to send payments to a web monetized site without using real money. 
 
 :::tip[Before you begin]
-You must have a [funded test wallet account](/guides/test-wallet-sign-up) on the Interledger Foundation's test network.
+You must have a [funded test wallet account](/guides/test-wallet-sign-up) on Interledger Foundation's test network.
 :::
 
 ## Install the extension
     
 <Steps>
     1. Sign in to your <LinkOut href="https://wallet.interledger-test.dev/">wallet account</LinkOut>.
-    2. Download and install the Interledger Foundation's [Web Monetization browser extension](/supporters/get-started#install-the-extension).
-    3. Enter the payment pointer associated with your test wallet and a budget amount, then click **Connect**.<br /><img
+    2. Download and install Interledger Foundation's [Web Monetization browser extension](/supporters/get-started#install-the-extension).
+    3. Enter the payment pointer associated with your test wallet. For example, `$ilp.interledger-test.dev/mytestwallet`. Then, enter a budget amount and click **Connect**.<br /><img
```

Comment:
I'm going to ask testnet team to replace payment pointer with wallet address now 😄🤞🏽. But looks good!